### DISCUSSION TITLE
Release v1.0.6 / 5.55.6 (pre-release)

### DISCUSSION THREAD

**Discussion by DavidXanatos** [sandboxie-plus/Sandboxie#1491] [Announcements]
![0 5 5](https://user-images.githubusercontent.com/3890945/105014988-575cb680-5a41-11eb-90d5-d942d28184b2.png)

This build fixed various issues with the previous build and adds some new functionality

#### You can support the project through donations, any help will be greatly appreciated.

[<img src="https://www.paypalobjects.com/en_US/AT/i/btn/btn_donateCC_LG.gif">](https://xanasoft.com/go.php?to=donate)

# ChangeLog

### Added
- replaced "Open with" with a Sandboxie dialog to work on Windows 10 [#1138](https://github.com/sandboxie-plus/Sandboxie/issues/1138)
- added ability to run Win32 store apps in App Compartment mode (on Windows 11 requires COM to be open)
-- Note: this does not mean UWP store apps, just regular win32 apps packaged to be deployed via the store
- added new debug options "UnstrippedToken=y" and "KeepUserGroup=y"
- added double click to recover files and folders in recovery window [#1466](https://github.com/sandboxie-plus/Sandboxie/issues/1466)
- added Ukrainian language on Plus UI (by SuperMaxusa) [#1488](https://github.com/sandboxie-plus/Sandboxie/pull/1488)

### Changed
- "UseSbieWndStation=y" is now the default behaviour [#1442](https://github.com/sandboxie-plus/Sandboxie/issues/1442)
- disabled Win32k hooking when HVCI is enabled due to an incompatibility (BSOD) [#1483](https://github.com/sandboxie-plus/Sandboxie/issues/1483)

### Fixed
- fixed box initialization issue in Privacy mode [#1469](https://github.com/sandboxie-plus/Sandboxie/issues/1469)
- fixed issue with shortcuts creation introduced in a recent build [#1471](https://github.com/sandboxie-plus/Sandboxie/issues/1471)
- fixed various issues in Privacy Enhanced boxes and rule specificity
- fixed issue with SeAccessCheckByType and alike
- fixed issues with Win32k hooking on 32 bit Windows [#1479](https://github.com/sandboxie-plus/Sandboxie/issues/1479)

### Removed
- removed obsolete SkyNet rootkit detection from 32 bit build

<hr /><em>This discussion was created from the release <a href='https://github.com/sandboxie-plus/Sandboxie/releases/tag/1.0.6'>Release v1.0.6 / 5.55.6</a>.</em>

**Comment by SbMan1** [comment]
This new 1.0.6 version is not letting me use Brave Beta or Brave Developer the standard way or with UAC Admin option. Can't click on bookmark to make it work and address won't let me type anything in. Previous version was Release v0.9.8d / 5.53.3 which worked fine for Brave.  Also Vivaldi Stable & Opera Stable, MS Edge Stable and Waterfox Current stable also all work. 
Noticed 1.0.x versions state Brave issues somewhere so decided to wait a bit.

I'm using DefaultBox for the above browsers. Once ran into Brave problem I then Deleted the DefaultBox and had same problem. Next day or two also same problem after rebooted a few times.

Thanks for all the hard work and updates, very much appreciated.

**Comment by SbMan1** [comment]
I missed that 1.0.6 was a Pre-release. So went to 1.0.5 and Brave works great now.  Thanks for your time !!
